app/controllers/users_controller.rb

```diff
@@ -1,5 +1,6 @@
 class UsersController < ApplicationController
   def myfeed
     @userposts = User.find(params[:id]).posts
+    @userid = User.find(params[:id]).id
```

Comment:
try to use camel case syntax for variables
`@user_id` instead of `@userid`
makes the reading much easier

also, just a suggestion, if you need an attribute of the user, consider if you should pass the `@user` object. this way you would have access to the user attributes and the user object in the html view.